lib/std/crypto/gimli.zig

```diff
@@ -249,15 +249,19 @@ pub const Aead = struct {
             in = in[State.RATE..];
             out = out[State.RATE..];
         }) {
-            for (buf[0..State.RATE]) |*p, i| {
-                p.* ^= in[i];
-                out[i] = p.*;
+            const d = in[0..State.RATE];
```

Comment:
Why is the loop split in two?

Reply:
Same as above. The second one becomes `movups` instead of being a byte by byte copy.

Reply:
The second loops have been replaced with `mem.copy()` for clarity (no binary code chang).